pkgs/top-level/python-packages.nix

```diff
@@ -5800,7 +5800,9 @@ self: super: with self; {
 
   javaproperties = callPackage ../development/python-modules/javaproperties { };
 
-  jax = callPackage ../development/python-modules/jax { };
+  jax = callPackage ../development/python-modules/jax {
+    inherit (pkgs.writers) writePython3Bin;
```

Comment:
I think it's OK to just take `writers` as argument (there prboably are precedents of doing so. Tbd: check)

Reply:
hmm i'm not able to find any precedent for that in `python-packages.nix`. i don't really have a preference though